kanin/src/app/task.rs

```diff
@@ -100,12 +105,14 @@ async fn handle_request<H, Args, Res>(
     let properties = req.properties().cloned();
     let reply_to = properties.as_ref().and_then(|p| p.reply_to().clone());
     let correlation_id = properties.as_ref().and_then(|p| p.correlation_id().clone());
+    let app_id = req.app_id().unwrap_or("<unknown>");
+
+    let handler_name = std::any::type_name::<H>();
+    info!("Received request on handler {handler_name:?} from {app_id}");
+
     // Call the handler with the request.
     let response = handler.call(&mut req).await;
-    debug!(
-        "Handler {:?} produced response: {response:?}",
-        std::any::type_name::<H>()
-    );
+    info!("Handler produced response: {response:?}");
```

Comment:
I removed the name of the handler here (it feels weird that it's printed twice when you can just use the `req_id` to deduct the name.

But I have a concern about printing the full response of every single request. For small responses it's no problem; but it doesn't seem right thing to do as the default for every service.

Reply:
```rs
info!("Handler produced response of {} bytes that will be published on {reply_to}", bytes_response.len());
```
could be an option.

Reply:
There won't always be a reply_to. But maybe you can log in the different scenarios of reply_to/no reply_to

Reply:
How about this? :)